src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutRecoverabilityBehavior.cs

```diff
@@ -7,6 +7,7 @@ namespace NServiceBus
     using System;
     using System.Collections.Generic;
     using System.Threading.Tasks;
+    using JetBrains.Annotations;
```

Comment:
And by the way, even though these attributes live in the `JetBrains.Annotations` namespace, we don't reference `JetBrains.Annotations.dll` or the NuGet package. The source code for these attributes lives in our repo. And this is OK.